bindings/pyroot/pythonizations/test/rdataframe_misc.py

```diff
@@ -1,4 +1,3 @@
-import cppyy
```

Comment:
Is this change related to the title of PR?

Reply:
No, that was just something that I tried out for the CI.

But now I understand that this "macos12soversion" patch is still necessary.

I propose however to still sync the `loader.py` file now that we already have taken a look. The PR was updated to do that.